dev-docs/bidders/adview.md

```diff
@@ -7,23 +7,24 @@ tcfeu_supported: true
 gvl_id: 1022
 usp_supported: true
 coppa_supported: true
+gpp_supported: true

```

Comment:
`gpp_supported` is deprecated. You have to be more specific on what you actually support

```suggestion
gpp_sids: // the sections you support. List of cfeu, tcfca, usnat, usstate_all, usp
```

see https://docs.prebid.org/dev-docs/bidder-adaptor.html

Reply:
ok, changed gpp_support to gpp_sids, pls check it again.